src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssCreate.java

```diff
@@ -97,9 +100,11 @@ public Response create(String destName, CreateParams params) throws Exception {
         ValidateUtils.checkNullParameter(params.getType() == null, "params type is null");
         ValidateUtils.checkNullParameter(params.getMode() == null, "params mode is null");
         ValidateUtils.checkNullParameter(destName == null, "destName is null");
+        ValidateUtils.checkIllegalParameter(params.getMode()
+                .matches("[r-][w-][w-][r-][w-][w-][r-][w-][w-]"), "params.mode must be of format 'rwxrwxrwx'");
```

Comment:
make the regex code performed in a helper private method in the class.. how about "x" in the match list. I dont see it.  You can write a method called isModeValidFormat(..) and return a boolean value.